content/guides/getting-started/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Getting started
+keywords: get started, getting started, how to get started
+description: Get started with Docker. You'll learn about Docker Desktop, developing with Docker, as well as how to build and push your first image.
+aliases:
+- /get-started/
+---
+
+This getting started guide is a series of quick topics that show you how to:
+
+* Install Docker Desktop
+* Develop with containers
+* Build and push your first image
+
+By the end of this guide, you'll have hands-on experience using Docker Desktop
+and a better understanding of the benefits of containerizing your applications.
+
```

Comment:
[Question] Is it worth mentioning the new name/URL for the original get started guide on this page? In case users come here looking for the workshop. Something like: 
```suggestion
If you want a deeper dive into getting started with Docker, see [Docker workshop](/guides/workshop/).
```
Or maybe that could be included in "What's next" page?

Reply:
We may possibly remove the workshop. I don't think it's supposed to be part of the new journey. A lot of the content is identical to the Docker concept topics, just in a different (longer) format.